os/arch/arm/src/amebad/Kconfig

```diff
@@ -67,4 +67,23 @@ config AMEBAD_TRUSTZONE
 	default n
 	select ARMV8M_TRUSTZONE
 
+config AMEBAD_RDP
+	bool "Enable Amebad RDP"
+	default n
+	select AMEBAD_RDP
```

Comment:
Cheah, this select is redundant, you can remove this.

Reply:
Thanks,  and sorry about that, i will recommit again.